staker/challenge-cache/cache_test.go

```diff
@@ -69,6 +74,148 @@ func TestCache(t *testing.T) {
 	}
 }
 
+func TestPrune(t *testing.T) {
+	ctx := context.Background()
+	basePath := t.TempDir()
+	if err := os.MkdirAll(basePath, os.ModePerm); err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
```

Comment:
because basePath was created as t.TempDir - t already has a cleanup stage deleting the baseDir with everything underneath it. No need for this extra cleanup